plane/src/database/connect.rs

```diff
@@ -95,9 +95,10 @@ async fn create_backend_with_key(
                 drone_id,
                 expiration_time,
                 allowed_idle_seconds,
-                last_keepalive
+                last_keepalive,
+                state
             )
-            values ($1, $2, $3, now(), $4, now() + $5, $6, now())
+            values ($1, $2, $3, now(), $4, now() + $5, $6, now(), $11)
```

Comment:
Is this supposed to be `$7` instead of `$11`?

Reply:
no, this is the value for `state`, which is the 11th argument in the list below. `$7` is `key.name`.